src/cli/src/cluster/install/local.rs

```diff
@@ -170,7 +171,7 @@ async fn launch_spu(spu_index: u16, client: SharedK8Client, option: &InstallComm
         .print();
 
     println!("SPU<{}> cmd: {:#?}", spu_index, cmd);
-
+    println!("SPU log generated at /tmp/spu_log_{}.log", spu_id);
```

Comment:
So logic duplicated here.  Can you assign location of the log then just reuse here?  for example:
```
let log = format!("/tmp/spu_log_{},spu_id);      
....
println!("SPU log generated at {}", log);
```